modules/ROOT/pages/keycloak/keycloak-setup.adoc

```diff
@@ -1,6 +1,6 @@
 //include::{partialsdir}/attributes.adoc[]
 
-= Keycloak Setup
+= Setting Up {keycloak-service}
```

Comment:
When I view this in the Identity Managment Service The heading is not at the same level as other headings. e.g The Binding a Mobile Client, I've included a screenshot 
![image](https://user-images.githubusercontent.com/6498727/42162611-3abbe682-7df7-11e8-83f3-a8e4a0537e12.png)

Reply:
However it works fine in the getting-started guide 
![image](https://user-images.githubusercontent.com/6498727/42162987-b9038148-7df8-11e8-985e-67739a6093b0.png)